src/app/ui/components/containers/footers/footer.tsx

```diff
@@ -2,20 +2,33 @@ import type { ReactNode } from 'react';
 
 import { Flex, styled } from 'leather-styles/jsx';
 
+type FooterVariant = 'page' | 'card' | 'popup';
 interface FooterProps {
   children: ReactNode;
-  variant?: 'page' | 'card';
+  variant?: FooterVariant;
   flexDirection?: 'column' | 'row';
 }
 
+function getMaxWidth(variant: FooterVariant) {
```

Comment:
I added this fix also to make sure the footer has the same max width as the content in the PSBT sign screen, otherwise it looks weird if you make the window size bigger

![Screenshot 2024-04-26 at 15 31 24](https://github.com/leather-wallet/extension/assets/2938440/98ce4080-544f-4b04-986b-5787dfba87af)